package.json

```
@@ -11,7 +11,7 @@
   ],
   "scripts": {
     "build": "fedx-scripts webpack",
-    "i18n_extract": "fedx-scripts formatjs extract",
+    "i18n_extract": "formatjs extract --format node_modules/@openedx/frontend-build/lib/formatter.js --ignore {src,plugins}/**/*.json --out-file ./temp/babel-plugin-formatjs/Default.messages.json -- {src,plugins}/**/*.js*",
```

I've checked the `fedx-scripts` and I think there's a better solution because the `fedx-scripts` accepts additional arguments and I'd like to ask you to try it first:
```suggestion
    "i18n_extract": ""i18n_extract": "fedx-scripts formatjs extract plugins/**/*.js*",
```

We might need to add quotes, I'm unsure whether bash is running here or not:

```suggestion
    "i18n_extract": ""i18n_extract": "fedx-scripts formatjs extract 'plugins/**/*.js*'",
```

Please check:

https://github.com/openedx/frontend-build/blob/c70ef4e0a5e8702f7144f36b1b23195b94f5e09a/bin/fedx-scripts.js#L71-L81

---

Yes, I checked when I open this PR.

That is why I included `./temp/babel-plugin-formatjs/Default.messages.json` because behind the scenes the `fedx-scripts` is using that.
In general the problem is that the `fedx-scripts` `formatjs` don't allow to add additional source folders. But probably this isn't the idea of `fedx-scripts` of having too much custom options, just a basic framework for different Open edX frontend packages.

So I think this resume the execution, where each step executes the next one:
1. `npm run i18n_extract`
2. `fedx-scripts formatjs extract` that is installed on `node_modules/@openedx/frontend-build/bin/fedx-scripts.js` that adds those [commonArgs](https://github.com/openedx/frontend-build/blob/c70ef4e0a5e8702f7144f36b1b23195b94f5e09a/bin/fedx-scripts.js#L71-L81) to the next process
4. `node_modules/@formatjs/cli/bin/formatjs extract --format node_modules/@openedx/frontend-build/lib/formatter.js --ignore src/**/*.json --out-file ./temp/babel-plugin-formatjs/Default.messages.json -- src/**/*.js*` but this command don't include the `plugins` folder where this repository has additional code.

So my proposal is to replace the `fedx-scripts formatjs extract` executing a lower level command directly that also includes the `plugins` folder.

If you execute this PR before and after you will an increase on the strings to be translated, `145` strings on the branch that I opened the PR.
```bash
$ npm run i18n_extract

> @edx/frontend-app-authoring@0.1.0 i18n_extract
> formatjs extract --format node_modules/@openedx/frontend-build/lib/formatter.js --ignore {src,plugins}/**/*.json --out-file ./temp/babel-plugin-formatjs/Default.messages.json -- {src,plugins}/**/*.js*

[@formatjs/cli] [WARN] [FormatJS CLI] Duplicate message id: "authoring.proctoring.alert.error", but the `description` and/or `defaultMessage` are different.
$ grep '"id"' temp/babel-plugin-formatjs/Default.messages.json | wc -l
1860
```

I can't make it run with your proposal:

package.json changed to:
```json
"i18n_extract": "fedx-scripts formatjs extract plugins/**/*.js*",
```

Execute and output:
```bash
$ npm run i18n_extract

> @edx/frontend-app-authoring@0.1.0 i18n_extract
> fedx-scripts formatjs extract plugins/**/*.js*

Running with resolved config:
/home/ibranco/projects/openedx/frontend-app-authoring/node_modules/@openedx/frontend-build/config/babel.config.js

[@formatjs/cli] [WARN] Error: Error processing file plugins/course-apps/calculator/package.json 
Debug Failure. Output generation failed
```

Yes, it passes the arguments, but because the fedx-scripts will add more arguments, then the real formatjs executed will be kind of different!